tests/test_env_setup.sh

```diff
@@ -2,17 +2,27 @@
 
 if [[ "$OSTYPE" == "linux-gnu" ]]; then
```

Comment:
Could we merge these two if statements for setting TEST_TESTS_DIR and TEST_CODECHECKER_PACKAGE_DIR environment variables?
If OSTYPE is not set the default value should be linux-gnu.